src/Grpc.Net.ClientFactory/Internal/GrpcHttpClientFactory.cs

```diff
@@ -35,13 +35,15 @@ internal class GrpcHttpClientFactory<TClient> : INamedTypedHttpClientFactory<TCl
         private readonly Cache _cache;
         private readonly IServiceProvider _services;
         private readonly ILoggerFactory _loggerFactory;
-        private readonly IOptionsMonitor<GrpcClientFactoryOptions> _optionsMonitor;
+        private readonly IOptionsMonitor<GrpcClientFactoryOptions> _clientFactoryOptionsMonitor;
+        private readonly IOptionsMonitor<GrpcChannelOptions> _channelOptionsMonitor;
```

Comment:
`GrpcChannelOptions` are currently separate from `GrpcClientFactoryOptions`. Could refactor to make `ChannelOptions` a property on `GrpcClientFactoryOptions`.